examples/modules/comparison/plot_3_compare_sorter_with_ground_truth.py

```diff
@@ -73,6 +85,12 @@
 ax1 = sns.swarmplot(data=perf2, x='measurement', y='value', ax=ax1)
 ax1.set_xticklabels(labels=ax1.get_xticklabels(), rotation=45)
 
+##############################################################################
+# The confusion matrix is also a good summary of the score
+# it have the same shape as agreement matrix but have an extra columns for FN
+# and an extra row for FP
+
```

Comment:
# The confusion matrix is also a good summary of the score as it has
# the same shape as agreement matrix, but it contains an extra column for FN
# and an extra row for FP